src/base/QXmppMessage.cpp

```diff
@@ -1073,6 +1077,36 @@ void QXmppMessage::setSpoilerHint(const QString &spoilerHint)
         d->isSpoiler = true;
 }
 
+///
+/// Determines whether this message is a \xep{0369}: Mediated Information eXchange (MIX) invitation message as defined by \xep{0407}: Mediated Information eXchange (MIX): Miscellaneous Capabilities.
+///
+/// \since QXmpp 1.5
```

Comment:
If you're targeting `1.4` (which is going to be the next release) with this, then please use that everywhere.